### ISSUE TITLE
Error in "yarn build:release" step for 6.4.x and 6.5.0

### ISSUE THREAD

**Issue by jrodrigop** [okta/okta-signin-widget#2605]
### Describe the bug

As documented in https://github.com/okta/okta-signin-widget#build-and-test-commands

Starting from a clean git clone and checkout 6.5.0 or any 6.4.x, then after installation fails at:
```
~/ws/wokta/okta-signin-widget$ yarn install
...
Done in 146.57s.
~/ws/wokta/okta-signin-widget$ yarn build:release
...
...
Running "exec:build-release" (exec) task
...
...
asset okta-sign-in.js 4.66 MiB [emitted] (name: main) 1 related asset
modules by path ./node_modules/ 1.15 MiB 901 modules
modules by path ./src/ 1.39 MiB
  modules by path ./src/v2/ 623 KiB 136 modules
  modules by path ./src/v1/ 680 KiB 119 modules
  modules by path ./src/util/ 81.9 KiB 21 modules
  modules by path ./src/widget/ 11.4 KiB 4 modules
  modules by path ./src/models/*.ts 23.1 KiB 2 modules
  2 modules
modules by path ./packages/ 1.23 MiB
  modules by path ./packages/@okta/courage-dist/esm/src/ 893 KiB 73 modules
  modules by path ./packages/@okta/courage-dist/esm/_virtual/*.js 7.05 KiB 26 modules
  modules by path ./packages/@okta/courage-dist/esm/lib/ 85.1 KiB 22 modules
  json modules 87.6 KiB 2 modules
  modules by path ./packages/vendor/ 67.1 KiB 2 modules
  ./packages/@okta/qtip2/dist/jquery.qtip.js 121 KiB [built] [code generated]
webpack 5.51.1 compiled successfully in 106423 ms
$ yarn clean:esm && yarn build:rollup-release && yarn lint:esm
$ rimraf target/esm
$ rollup -c rollup.release.config.js
>>
>> src/index.ts → ./target/esm...
>> (!) Circular dependency
>> src/v2/view-builder/internals/index.js -> src/v2/view-builder/internals/BaseOktaVerifyChallengeView.js -> src/v2/view-builder/internals/index.js
>> created ./target/esm in 21.4s
$ yarn eslint --no-ignore --no-eslintrc --no-inline-config --config scripts/buildtools/eslint/.eslintrc.esm.js `find target/esm -type f -name "*.js"`
$ C:\Users\rodusr\ws\wokta\okta-signin-widget\node_modules\.bin\eslint --no-ignore --no-eslintrc --no-inline-config --config scripts/buildtools/eslint/.eslintrc.esm.js `find target/esm -type f -name *.js`
>> Invalid option '-t' - perhaps you meant '-c'?
>> error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
>> error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
>> error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
>> Exited with code: 2.
>> Error executing child process: Error: Process exited with code 2.
Warning: Task "exec:build-release" failed. Use --force to continue.

Aborted due to warnings.


Execution Time (2022-07-14 19:28:00 UTC-5)
exec:clean                3.2s  ██ 1%
exec:generate-config      2.3s  █ 1%
exec:build-types         26.5s  ██████████ 12%
postcss:minify            2.8s  ██ 1%
exec:build-release    2m 55.4s  █████████████████████████████████████████████████████████████████ 82%
Total 3m 34.7s

error Command failed with exit code 3.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

### What is expected to happen?

Command should complete without errors as previous versions (tested in 6.3.5)

### What is the actual behavior?

Build process is aborted

### Reproduction Steps

1. clone repo
2. checkout version 6.5.0 or 6.4.x
3. run `yarn install`
4. run `yarn build:release`

### SDK Versions

6.4.x and 6.5.0

### Execution Environment

Using Windows 10:
```
~/okta-signin-widget$ node --version
v14.19.1
~/okta-signin-widget$ npm --version
6.14.16
~/okta-signin-widget$ yarn -version
1.22.18
~/okta-signin-widget$ git status
HEAD detached at okta-signin-widget-6.5.0
nothing to commit, working tree clean
```

**Comment by denysoblohin-okta** [comment]
What kind of CLI do you use to build in Windows 10? Linux Bash Shell, Cygwin, WSL?

**Comment by jrodrigop** [comment]
Still failing in 6.7.0

**Comment by denysoblohin-okta** [comment]
Internal ref: OKTA-529953

**Comment by jrodrigop** [comment]
can you please update on this one? I see there is a potential fix, but code is not merged? this is still failing in V7.2.1

**Comment by jrodrigop** [comment]
any update?
